system-variables.md

```diff
@@ -1450,18 +1450,29 @@ SET tidb_query_log_max_len = 20
 
 ### tidb_restricted_read_only <span class="version-mark">New in v5.2.0</span>
 
+> **Warning:**
+>
+> For TiDB versions earlier than v5.3.1, this variable has a defect and using it might cause unexpected results. Ensure that you only use this variable in TiDB versions later than v5.3.1.
```

Comment:
```suggestion
>For TiDB versions earlier than v5.3.1 or v5.4.1, this variable has a defect and using it might cause unexpected results. Ensure that you use this variable ONLY in TiDB versions later than v5.3.1 or v5.4.1.
```